c2corg_ui/templates/base.html

```diff
@@ -60,7 +60,7 @@
     <div class="page-header">
       <button class="btn btn-default menu-open-close header"><span class="glyphicon glyphicon-align-justify"></span></button>
       <div class="quick-search">
-        <app-simple-search dataset="wrcb"></app-simple-search>
+        <app-simple-search dataset="wrcbx"></app-simple-search>
```

Comment:
To be discussed/confirmed, but as for outings, I am not sure we want to have the xreports in the simple search.